compiler/dcalc/from_scopelang.ml

```diff
@@ -589,20 +589,21 @@ let translate_rule
   match rule with
   | S.ScopeVarDefinition { var; typ; e; _ }
   | S.SubScopeVarDefinition { var; typ; e; _ } ->
+    let scope_var, _ = var in
+    let decl_pos = snd (ScopeVar.get_info scope_var) in
```

Comment:
We prefer to use `Mark.get` / `Mark.remove` for the pairs defined as `Mark.ed` ; although we do match them as pairs very often (doing differently would be extremely verbose given how pervasive this is), this is still more descriptive than the built-ins `fst`/`snd`.

Reply:
Agreed, fixed.